pysat/_instrument.py

```diff
@@ -1497,9 +1475,6 @@ def load(self, yr=None, doy=None, date=None, fname=None, fid=None,
                 self.meta = self._curr_meta.copy()
             else:
                 self.data = self._null_data.copy()
-                # line below removed as it would delete previous meta, if any
-                # if you end a seasonal analysis with a day with no data, then
-                # no meta: self.meta = pysat._meta.Meta()
```

Comment:
I responded via email, GitHub placed the comment. The system also won't allow me to offer a direct suggestion on deleted lines. 
```
                # If a new default/empty Meta is added here then it creates
                # a bug. If a seasonal or other multi-day analysis ends
                # on a day with no data then there is no variable metadata
                # for the user to work with. Do not re-introduce this issue.
```

Reply:
This bad location was my fault - unintended consequence of trying to get to suggestion thing to work.